src/util-base64.c

```diff
@@ -103,7 +103,6 @@ uint32_t DecodeBase64(uint8_t *dest, const uint8_t *src, uint32_t len) {
             /* Invalid character found, so decoding fails */
             if (src[i] != '=') {
                 valid = 0;
-                numDecoded = 0;
```

Comment:
I wonder if it would make sense to support the partial decoding there as well. Any thoughts @regit ?